pkg/certificates/service.go

```diff
@@ -0,0 +1,86 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package certificates
+
+import (
+	"context"
+
+	"go.uber.org/zap"
+	"google.golang.org/grpc/peer"
+
+	"storj.io/storj/pkg/certificates/authorizations"
+	"storj.io/storj/pkg/identity"
+	"storj.io/storj/pkg/pb"
+)
+
+// Certificates implements pb.CertificatesServer
+type Certificates struct {
```

Comment:
It looks that this file should be more `endpoint.go` and this struct `Endpoint`.

Reply:
Yea, you're right.